src/core/qgismobileapp.h

```diff
@@ -40,7 +40,7 @@
 #if defined( Q_OS_ANDROID )
```

Comment:
I think these platform specific imports should no(t) (longer) be required

Reply:
done let's see